extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfigPersistenceUnit.java

```diff
@@ -329,9 +329,18 @@ public enum NullOrdering {
         @ConfigItem(defaultValue = "none")
         public NullOrdering defaultNullOrdering;
 
+        /**
+         * Improve statement caching for heavily used IN clause queries
+         *
+         * @asciidoclet
+         */
+        @ConfigItem(defaultValue = "true")
```

Comment:
From what I can see, you are changing the default behavior here. I'm not sure if we want to do that.

@Sanne any opinion on this?

Reply:
I'm not sure - it's not particularly dangerous, might be fine and I agree it's often a good idea to enable it.

Wondering if we should switch the default in ORM6 (it's still false currently), I'll as the team.

I'm inclined to enable it.

Reply:
How about declaring this attribute as `Optional<Boolean>` and set it only if it's explicitly set?

Reply:
Given we are leaning towards making the default true, that wouldn't make much sense.